src/context.rs

```diff
@@ -63,28 +67,49 @@ impl Context {
         }
     }
 
+    /// Increases the reference count on self by one. This prevents self from being destroyed
+    /// until a matching call to drop() is made.
+    /// 
+    /// The number of references to a cairo_t can be get using Context::get_reference_count().
```

Comment:
This probably wants to be `#[doc(hidden)]` or put onto the `glib::translate` rails.